src/tools/crawl.ts

```diff
@@ -4,10 +4,11 @@
 
 // tslint:disable: no-console
 
-import fg from 'fast-glob';
 import * as fs from 'fs';
 import * as path from 'path';
 
+const fg = require('fast-glob');
```

Comment:
nit: was this supposed to change?

Reply:
Yeah, I did this as a hack to make TS happy after turning off esModuleInterop, which isn't otherwise necessary.